pkg/zcache/local_cache.go

```diff
@@ -5,20 +5,27 @@ import (
 	"encoding/json"
 	"errors"
 	"fmt"
+	"time"
+
 	"github.com/allegro/bigcache/v3"
 	"github.com/zondax/golem/pkg/metrics"
 	"github.com/zondax/golem/pkg/metrics/collectors"
 	"go.uber.org/zap"
-	"time"
 )
 
 const (
-	neverExpires          = -1
-	errorTypeLabel        = "error_type"
-	cleanupErrorMetricKey = "localCacheCleanupErrors"
-	iterationErrorLabel   = "iteration_error"
-	unmarshalErrorLabel   = "unmarshal_error"
-	deletionErrorLabel    = "deletion_error"
+	neverExpires = -1
+
+	errorTypeLabel         = "error_type"
+	itemCountLabel         = "cleanup_item_count"
+	residentItemCountLabel = "resident_item_count"
+	deletedItemCountLabel  = "deleted_item_count"
+	iterationErrorLabel    = "iteration_error"
+	unmarshalErrorLabel    = "unmarshal_error"
+	deletionErrorLabel     = "deletion_error"
+
+	cleanupItemCountMetricKey = "localCacheCleanupItemCount"
+	cleanupErrorMetricKey     = "localCacheCleanupErrors"
```

Comment:
Let's stick to what we currently use as standard for metrics names. We are currently using snake case, instead of camel case. Otherwise we will have a messy mix of metrics

Reply:
Alright, though the cleanupErrorMetricKey was always camel case: https://github.com/Zondax/golem/blob/aeff87b97786bd29953049da8131222eea8bee51/pkg/zcache/local_cache.go#L18 
Maybe it's camelCase for metric keys and snake_case for labels ?

Reply:
That is another one to change... take a loot at zrouter. We use snake case there